src/main/java/io/github/jhipster/registry/web/rest/AccountResource.java

```diff
@@ -55,6 +55,8 @@ public String isAuthenticated(HttpServletRequest request) {
             } else if (authentication.getPrincipal() instanceof String) {
                 login = (String) authentication.getPrincipal();
                 log.debug("The username `{}` has been found using OpenID Connect", login);
```

Comment:
This line doesn't seem accurate since OpenID Connect only relates to the following if statement.

Reply:
I wasn't aware about its purpose. If it's not used in any other flow, then, yes, it can be removed. Do you want me to take this up or you can do that in your branch?

Reply:
I'll remove it since it doesn't seem necessary.